### PR TITLE
Features/wadnr 1969 add county geospatial area index page

### PR DESCRIPTION
**Pull request by Adrickle**

### CODE REVIEW

File: Source/ProjectFirma.Web/ProjectFirma.Web.csproj [hunk 1]
```diff
@@ -707,8 +710,13 @@
     <Compile Include="Models\Generated\WashingtonLegislativeDistrict.Binding.cs" />
     <Compile Include="Models\Generated\WashingtonLegislativeDistrict.DatabaseContextExtensions.cs" />
     <Compile Include="Models\Generated\WashingtonLegislativeDistrictPrimaryKey.Binding.cs" />
+<<<<<<< HEAD
```
Comment (tomkamin) [inline Source/ProjectFirma.Web/ProjectFirma.Web.csproj:713]: Looks like the conflict here wasn't completely resolved

File: Source/ProjectFirma.Web/Views/County/Detail.cshtml [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+            <div class="panel-heading panelTitle">
+                <h2>@FieldDefinition.Project.GetFieldDefinitionLabelPluralized()</h2>
+            </div>
+            <div class="panel-body">
+                @Html.DhtmlxGrid(ViewDataTyped.BasicProjectInfoGridSpec, ViewDataTyped.BasicProjectInfoGridName, ViewDataTyped.BasicProjectInfoGridDataUrl, "height:300px", DhtmlxGridResizeType.VerticalResizableHorizontalAutoFit)
+            </div>
+        </div>
+    </div>
+</div>
+@*TODO Neutered Per WA DNR #1446. May decide to bring it back later
```
Comment (tomkamin) [inline Source/ProjectFirma.Web/Views/County/Detail.cshtml:101]: we might be able to rip this comment out and the JS dealing with switching tabs.

**Review by tomkamin — CHANGES_REQUESTED** [review]

File: Source/ProjectFirma.Web/ProjectFirma.Web.csproj (hunk 1 above)
Comment (Adrickle) [reply Source/ProjectFirma.Web/ProjectFirma.Web.csproj]: done

File: Source/ProjectFirma.Web/Views/County/Detail.cshtml (hunk 2 above)
Comment (Adrickle) [reply Source/ProjectFirma.Web/Views/County/Detail.cshtml]: I took out all the tab stuff and the comment

**Review by tomkamin — APPROVED** [review]

**Pull request merged**